src/client/actions/lists.js

```diff
@@ -0,0 +1,78 @@
+import 'whatwg-fetch';
+import _ from 'lodash';
+import { receivedTask } from './tasks';
+import { fetched, fetching } from './spinner';
+import { requestJSON } from '../helpers';
+
+require('isomorphic-fetch');
```

Comment:
import fetch from 'isomorphic-fetch';
(require is commonJS, webpack is smart but it's better to not mix standards)